### DISCUSSION TITLE
Has copilot been removed from student pack?

### DISCUSSION THREAD

**Discussion by paulzhn** [community/community#78526] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I received an email suddenly, which said:

> Your GitHub Copilot access will expire soon
>  
> GitHub Copilot
> Hi there,
> 
> Thank you for using your free subscription to GitHub Copilot. Your coupon is set to expire on 2025-03-07. No further steps are needed on your end.
> 
> We hope you enjoy using GitHub Copilot and participating in the developer community.
> 
> 
> 

I think my student verification is valid, and I found that the copilot is not available on https://education.github.com/pack/offers, but only a primer for it. Has the copilot been removed from the student pack?

**Comment by IgnacioPalma** [comment, marked as answer]
Hello @paulzhn! :wave:

GitHub Copilot is available for free for students by the Student Developer Pack, although it doesn't appear in the page :sweat_smile:
So no, it doesn't have been removed from the pack :D

**Comment by kishnae** [comment]
hello same here shifted to windows from ubuntu  and as I scynced my data found that I needed to login github copilot 
as I loggged in payment gateway page opened 
![image](https://github.com/community/community/assets/106668895/77824096-9dff-4d81-b227-5a34137a183e)

**Reply by davecheney** [reply]
@kishnae your student-2022 coupon expired last week. If you are still an active student you can apply for a new student developer pack which will provide another 2 years of copilot access.

**Comment by tinnamchoi** [comment]
![image](https://github.com/community/community/assets/101849050/59557eca-9360-4327-a8cd-11e11a2450f2)
![image](https://github.com/community/community/assets/101849050/0521ebaa-ac53-47e8-95f9-10f30e6fceca)
![image](https://github.com/community/community/assets/101849050/986c3037-46be-465d-b466-c59d580bb0be)

Same issue here. Submitted a new application for GitHub Education after the one from 2022 expired. It should be approved but I still can't use copilot.
